src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/EspressoPage.xaml

```diff
@@ -0,0 +1,182 @@
+                            LargeChange="5"/>
+                    <muxc:NumberBox x:Uid="Espresso_TemporaryKeepAwake_Minutes"
+                            Value="{Binding Minutes, Mode=TwoWay}"
+                            Minimum="0"
+                            SpinButtonPlacementMode="Compact"
+                            HorizontalAlignment="Left"
+                            Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                            MinWidth="90"
+                            IsEnabled="{Binding Mode, Converter={StaticResource EspressoModeToReverseBoolConverter}}"
+                            SmallChange="1" 
+                            LargeChange="5"/>
+                </StackPanel>
+            </StackPanel>
+        </StackPanel>
+
+        <RelativePanel x:Name="SidePanel" 
+                    HorizontalAlignment="Left"
+                    Width="{StaticResource SidePanelWidth}"
+                    Grid.Column="1">
+            <StackPanel x:Name="DescriptionPanel">
+                <TextBlock x:Uid="About_Espresso"
+                           x:Name="AboutTitle"
+                           Grid.ColumnSpan="2"
+                           Style="{StaticResource SettingsGroupTitleStyle}"
+                           Margin="{StaticResource XSmallBottomMargin}"/>
+                <TextBlock x:Uid="Espresso_Description"
+                           TextWrapping="Wrap"
+                           Grid.Row="1" />
+            </StackPanel>
+
+            <Border x:Name="AboutImage"
+                    CornerRadius="4"
+                    Grid.Row="2"
+                    MaxWidth="240"
+                    HorizontalAlignment="Left"
+                    Margin="{StaticResource SmallTopBottomMargin}"
+                    RelativePanel.Below="DescriptionPanel">
+                <HyperlinkButton x:Uid="Espresso_ImageHyperlinkToDocs">
+                    <Image x:Uid="Espresso_Image" Source="ms-appx:///Assets/Modules/ColorPicker.png" />
+                </HyperlinkButton>
+            </Border>
+            <StackPanel x:Name="LinksPanel"
+                        Margin="0,1,0,0"
+                        RelativePanel.Below="AboutImage"
+                        Orientation="Vertical" >
+                <HyperlinkButton x:Uid="Espresso_ImageHyperlinkToDocs">
+                    <TextBlock x:Uid="Module_overview" />
+                </HyperlinkButton>
+                <HyperlinkButton NavigateUri="https://aka.ms/powerToysGiveFeedback">
+                    <TextBlock x:Uid="Give_Feedback" />
+                </HyperlinkButton>
+
+                <TextBlock 
+                x:Uid="AttributionTitle"
+                Style="{StaticResource SettingsGroupTitleStyle}" />
+
+                <HyperlinkButton Margin="0,-3,0,0"
+                                NavigateUri="https://github.com/microsoft/PowerToys/">
+                    <TextBlock Text="Den Delimarsky's Espresso" TextWrapping="Wrap" />
```

Comment:
```suggestion
                    <TextBlock Text="Den Delimarsky's Espresso implementation" TextWrapping="Wrap" />
```
Sounds better imo. Btw this should be localized.

Reply:
> Disagree with the wording. This is the name of the tool, and not an implementation of something like an "Espresso API," hence the "[...] implementation" suffix is not necessary.

Okay, I see your point. Agree.


> Re: localization of the string - I've seen in other modules the string being included directly in the XAML, so I am not even sure if we are localizing them. That said, I do agree that as a best practice, the strings should be contained within the same `Resources.resw` file. I've just added that with 5cf82d7000cd6f52835af59aa37d6632159ecd39.

I opened an issues regarding this for ColorPicker's page. Let's wait the feedback on the issue.